src/opnsense/mvc/app/models/OPNsense/Monit/Monit.php

```diff
@@ -315,39 +315,6 @@ public function performValidation($validateFullModel = false)
         return $messages;
     }
 
-    /* return the dirty marker file location */
-    private function dirtyFile()
-    {
-        return (new AppConfig())->application->tempDir . '/monit.dirty';
```

Comment:
since I refactored this recently it means you can remove use AppConig line at the beginning of the file